pkg/encryptedsecret/check_right.go

```diff
@@ -0,0 +1,46 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build !windows
+
+package encryptedsecret
+
+import (
+	"fmt"
+	"os/user"
+	"syscall"
+)
+
+func checkRights(path string) error {
```

Comment:
Good point, haven't thought about that. That behavior is not supported in the way that the `python` binary will never belong to the `dd-agent` user or have the correct rights.

We should add a section to the doc explaining that they have to use shebang for their script instead of passing the script as an argument to `python`.